docs/concepts/workloads/controllers/deployment.md

```diff
@@ -43,6 +43,7 @@ In this example:
 
 * A Deployment named `nginx-deployment` is created, indicated by the `metadata: name` field.
 * The Deployment creates three replicated Pods, indicated by the `replicas` field.
+* Set the `spec.selector` field of a Deployment to match the labels of its `.spec.template.metadata.labels`. In this case the label is `app: nginx`.
```

Comment:
I've talked to users who were confused by the use of the word "match" when talking about selectors. It sounds like `selector.matchLabels` has to be exactly equal to `template.metadata.labels`, which is not true.

What about:

```markdown
* The `selector` field defines how the Deployment finds which Pods to manage.
  In this case, we simply select on one label defined in the Pod template (`app: nginx`).
  However, more sophisticated selection rules are possible,
  as long as the Pod template itself satisfies the rule.
```

Reply:
@enisoc Yes, I think this comment clear better, this avoids the user confused.